terra/models.py

```diff
@@ -239,6 +239,11 @@ def total_funding(self):
     def vacation_days(self):
         return sum([v.vacation_days() for v in self.vacation_set.all()])
 
+    def total_days_out(self):
+        vacation_days = sum([v.vacation_days() for v in self.vacation_set.all()])
```

Comment:
@aprigge This replicates logic from the `vacation_days()` function above.  This should call that function instead.